examples/language/gpt/model_zoo.py

```diff
@@ -14,13 +14,13 @@ def __init__(self,
                  checkpoint=False):
         super().__init__()
         self.checkpoint = checkpoint
-        self.model = GPT2LMHeadModel(
-            GPT2Config(n_embd=hidden_size,
-                       n_layer=num_layers,
-                       n_head=num_attention_heads,
-                       n_positions=max_seq_len,
-                       n_ctx=max_seq_len,
-                       vocab_size=vocab_size))
+        self.config = GPT2Config(n_embd=hidden_size,
```

Comment:
why n_embd =  hidden_size? it doesn't make sense.

Reply:
... n_embd = hidden_size in HF GPT models.